cypress/integration/login.js

```diff
@@ -21,7 +21,9 @@ describe('Sourcebooks login', function() {
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Demo User');
+        cy.get('.user-info__title').contains('Dovile Martinkute');
         cy.get('.main-nav').find('li').should('have.length', 1);
+        
+        cy.get('.calendar__date').contains(new Date().getDate());

```

Comment:
Good stuff! One little thing you could do is update the name of the test case so it reflects your day assertion, e.g. "... and check if today's date is selected" :)